### ISSUE TITLE
[Bug] monday.storage.instance.setItem does have wrong types

### ISSUE THREAD

**Issue by eidermaen** [mondaycom/monday-sdk-js#83]
**Describe the bug**
For `monday.storage.instance.setItem` there seem to be wrong types.

**Steps to reproduce**
1. Embed the SDK in an app
2. Call `monday.storage.instance.setItem`
3. Check the parameter types that can be passed to the method
    ![grafik](https://user-images.githubusercontent.com/74467184/229429725-3f543656-a5eb-4bea-b48a-12ea1ab88eea.png)
4. Also check implemented types: [client-data.interface.ts ](https://github.com/mondaycom/monday-sdk-js/blob/master/types/client-data.interface.ts#L72)

**Expected behavior**
According to the [documentation](https://developer.monday.com/apps/docs/mondaystorage) and the [implementation](https://github.com/mondaycom/monday-sdk-js/blob/master/src/client.js#L105) of the storage, it should be possible to also pass additional options like the `previous_version` attribute. Those types are not being reflected correctly in TS.
So I would assume the method to work like:

```ts
monday.storage.instance.setItem('some_key', myJsonData, {previous_version: previousVersion})
```

**Context and environment**
Tested using the sdk on client side in a widget.

**Comment by zivnadel** [comment]
I see that what you're stating is shown in one of the examples in the docs (in the Versioning example), but not in the methods tables. I could add the missing parameter, but I can't find a full list of the available options, and I don't want it to be incomplete. Is there a list buried somewhere with the possible options to be passed to `setItem`?

**Comment by zivnadel** [comment]
@yuhgto can you take a look at this? Do you have the `options` documented somewhere?

**Comment by eidermaen** [comment]
Hi @zivnadel,
I just got the confirmation from the monday crew that you can only pass `previous_version` inside the `options`. So should be fine to go only with this one attribute 👍🏻

**Comment by zivnadel** [comment]
@eidermaen the 'previous_version' is a string, right?

**Comment by eidermaen** [comment]
@zivnadel Correct, at least what I could see from the response object when running the code.

**Comment by zivnadel** [comment]
Fixed in #87
